lib/omise/OmiseCapabilities.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+class OmiseCapabilities extends OmiseApiResource
+{
+    const ENDPOINT = 'capability';
+
+    /**
+     * @var array  of the filterable keys.
+     */
+    const FILTERS = array(
+        'backend' => array('currency', 'type', 'chargeAmount')
+    );
+
+    protected function __construct($publickey = null, $secretkey = null)
+    {
+        parent::__construct($publickey, $secretkey);
+        $this->setupFilterShortcuts();
+    }
+
+    /**
+     * Retrieves capabilities.
+     *
+     * @param  string $publickey
+     * @param  string $secretkey
+     *
+     * @return OmiseCapabilities
+     */
+    public static function retrieve($publickey = null, $secretkey = null)
+    {
+        return parent::g_retrieve(get_class(), self::getUrl(), $publickey, $secretkey);
+    }
+
+    /**
+     * (non-PHPdoc)
+     *
+     * @see OmiseApiResource::g_reload()
+     */
+    public function reload()
+    {
+        parent::g_reload(self::getUrl());
+    }
+
+    /**
+     * Retrieves array of payment backends. Optionally pass in as many filter functions as you want
+     * (muliple arguments, or a single array)
+     *
+     * @param [func1,fun2,...] OR func1, func2,...
+     *
+     * @return array
+     */
+    public function getBackends()
+    {
+        $backends = array_map(
+            function ($backend) {
+                $id = array_keys($backend)[0];
+                $backend[$id]['_id'] = $id;
+                return (object)reset($backend);
```

Comment:
@jonrandy is there any reason you want to cast it to `Object`? 

Just I found it's a bit off that the following code won't give the same result (it suppose to).
```php
$capabbilities = OmiseCapabilities::retrieve();
print_r($capabbilities['payment_backends']);
```
and
```php
$capabbilities = OmiseCapabilities::retrieve();
print_r($capabbilities->getBackends());
```

Reply:
Why is it off? the `getBackends` returns backend `objects` created from the arrays in the JSON. Objects are nice to deal with in the code

Reply:
Just slight of feeling that it should be the same value & structure (especially that `_id` attribute can be found only when call `getBackends()`)

For casting to Object < ok